### ISSUE TITLE
Idea for feature request - "alert-me-at"

### ISSUE THREAD

**Issue by ralt** [stumpwm/stumpwm-contrib#38]
Hi,

I made a small command that I deem useful enough to be part of stumpwm-contrib, but am not sure if that's the opinion of maintainers. (It *is* pretty small, after all.)

The "alert-me-at" command will give you a remainder of something at some point in time. This is useful if you have a meeting at 3pm, it's 11am and you don't want to forget about it.

Future improvements could be adding "in 10 minutes" options, better handling errors, refactoring the repetitions in the code, etc.

Basically, the function is here: https://gist.github.com/ralt/a1c41a4a56399897a63c

Would stumpwm-contrib be open for a PR including this new module?

Cheers,
Florian

**Comment by dbjergaard** [comment]
Please implement as PR following the format of the other modules. I'll merge it as soon as I get the notification :)

**Comment by dbjergaard** [comment]
I think there are timers in stumpwm, is this different?

**Comment by ralt** [comment]
Hm, I haven't seen them. I'll look for them and report.

**Comment by dbjergaard** [comment]
https://stumpwm.github.io/git/stumpwm-git_12.html#Timers

**Comment by ralt** [comment]
I see:

```lisp
(defun run-with-timer (secs repeat function &rest args)
  "Perform an action after a delay of SECS seconds.
Repeat the action every REPEAT seconds, if repeat is non-nil.
SECS and REPEAT may be reals.
The action is to call FUNCTION with arguments ARGS."
```

So... it sounds related, although not exactly the same. stumpwm's timers could be used instead of using `(loop do (sleep 5))`, for example.

What should I do?

**Comment by dbjergaard** [comment]
The more contrib modules the merrier I say! Open a PR following the format and I'd be happy to merge it. Bonus points if your readme has all the required documentation and an example.
